sources_test.go

```diff
@@ -111,3 +113,22 @@ func TestArgs(t *testing.T) {
 	}
 
 }
+
+func TestGet(t *testing.T) {
+	defer gock.Off()
+
+	gock.New("http://localhost:8000").
```

Comment:
Can we add a few more test cases here?

Reply:
Such as? This test fully tests the codepath thus far; I see no value in testing anything else.

Reply:
There are at least a few things I can think of; redirects and 404 errors, for example. Does Get() follow 3xx redirects?